### DISCUSSION TITLE
Explore to dashboard workflow: **Via Library Panels**

### DISCUSSION THREAD

**Discussion by samjewell** [grafana/grafana#45864] [Explore]
See https://github.com/grafana/grafana/issues/20245#issuecomment-1049687362

When I've got an explore that I want to save and/or share with other people, or later add to dashboards, I'd like to persist it firstly as a Library Panel.

Also see the Slack thread here (for a comparison with Looker):
https://raintank-corp.slack.com/archives/C036J5B39/p1645714306459729

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!

**Reply by gelicia** [reply]
Migrated to an issue:  #83121
